docs/retrieve.txt

```diff
@@ -0,0 +1,342 @@
+.. _laravel-retrieve:
+
+==============
+Retrieve Data
+==============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: find one, find many, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+collections by using **find operations**. Find operations are commands that
+retrieve documents from the server.
+
+This guide includes the following sections:
+
+- :ref:`laravel-retrieve-setup`
+- :ref:`laravel-retrieve-matching`
+- :ref:`laravel-modify-find`
+- :ref:`laravel-addtl-info`
+
+.. _laravel-retrieve-setup:
+
+Prerequisites
+-------------
+
+Complete the :ref:`Quick Start <laravel-quick-start>` tutorial before running the examples in this
+guide.
```

Comment:
Q: just checking that you know this link doesnt work right now

Reply:
yeah, just merged the changes from Chris' PR though so should work now!